src/redis/BetterSocialCronQueue.js

```diff
@@ -64,7 +64,7 @@ class BetterSocialCronQueue extends BetterSocialQueue {
     }
 
     if (flag === 'dailyDeleteExpiredPost') {
-      return deleteExpiredPostProcess(job, done, queueInjection?.deleteActivityProcessQueue);
+      return deleteExpiredPostProcess(job, done, queueInjection?.removeActivityQueue);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line has been updated to use the new queue name `removeActivityQueue`. Make sure that the `deleteExpiredPostProcess` function is compatible with this new queue and that it's expecting the correct type of queue.
```diff
- return deleteExpiredPostProcess(job, done, queueInjection?.deleteActivityProcessQueue);
+ return deleteExpiredPostProcess(job, done, queueInjection?.removeActivityQueue);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->